src/segmentation/test.py

```diff
@@ -0,0 +1,66 @@
+import sys
+from argparse import ArgumentParser
+
+import wandb
+
+import torch
+from dataset import FiberDataset
+
+import onnx
+import onnxruntime
+
+
+def parse_args(raw_args):
+    parser = ArgumentParser(description="Evaluate a UNet model (onnx).")
+    parser.add_argument("--model_version", type=str, default="latest")
+    parser.add_argument("--test_size", type=int, default=255)
+    args = parser.parse_args(raw_args)
+    return args
+
+
+def to_numpy(tensor):
+    return (
+        tensor.detach().cpu().numpy() if tensor.requires_grad else tensor.cpu().numpy()
+    )
+
+
+if __name__ == "__main__":
+
+    run = wandb.init(
+        project="FiberDiameter",
+        job_type="test",
+        config=parse_args(sys.argv[1:]),
+    )
+
+    cfg = run.config
+
+    artifact = run.use_artifact(
+        "warm-kanelbullar/FiberDiameter/rendered-fibers-medium:latest"
+    )
+
+    dataset_path = artifact.download()
+
+    model_artifact = run.use_artifact(f"warm-kanelbullar/FiberDiameter/model.onnx:{cfg.model_version}")
```

Comment:
It's so nice that you are already using W&B models. can we rename it to seg_model? Mostly because maybe later we want to add different models for different tasks like the depth prediction for selecting 10 random points.

Reply:
perfect, will do :)